packages/editor/src/components/media-placeholder/index.js

```diff
@@ -127,7 +127,7 @@ class MediaPlaceholder extends Component {
 					{ __( 'Upload' ) }
 				</FormFileUpload>
 				<MediaUpload
-					gallery={ multiple }
+					gallery={ multiple && type === "image" }
```

Comment:
Looks like there's an eslint issue here. We should use single quotes. You can test locally using `npm run lint` or install an eslint plugin for your IDE.